spec/features/cookie_banner_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+RSpec.feature 'Cookie banner', js: true do
```

Comment:
nit: line break between requires and class declaration
```suggestion

RSpec.feature 'Cookie banner', js: true do
```

Reply:
In general though, nice test! Glad to see that js testing was possible here.

Reply:
done